test/cli_inspect_test.go

```diff
@@ -68,7 +73,7 @@ func (suite *PouchInspectSuite) TestInspectCreateAndStartedFormat(c *check.C) {
 	// Inspect LogPath, LogPath should not be empty after container's start.
 	// by default, the container has log type of json-file.
 	output = command.PouchRun("inspect", "-f", "{{.LogPath}}", name).Stdout()
-	expectedLogPath := fmt.Sprintf("/var/lib/pouch/containers/%s/json.log", containerID)
```

Comment:
I search in pouch , find more hard code `/var/lib/pouch`, maybe we can fix in this pr to replace it use this func?

Reply:
I can't find anyone in testcase....Can you give a case?

Reply:
I mean in whole project can do as the same way, not only the testcase.

Reply:
I've searched the whole project. Actually, `/var/lib/pouch` occurred in 5 go file. 
One of them is the default config of pouchd. Two of them is the default config of storage.

Two of them inside the test.
First one, a mock test, acceptable.
https://github.com/alibaba/pouch/blob/1f7dbf60d406210c65fb96ea4682f6d04f97773e/client/system_info_test.go#L40

Second one, a hard code path in cli test.
https://github.com/alibaba/pouch/blob/638b38fa9a09ae52922a900d611817b69e017d7c/test/cli_inspect_test.go#L71
I think this one could use `GetRootDir` function for more generality.

@ZYecho @rudyfly

Reply:
LGTM @zhuangqh thanks for your check.